helix-term/src/application.rs

```diff
@@ -53,32 +62,39 @@ pub struct Application {
     signals: Signals,
     jobs: Jobs,
     lsp_progress: LspProgressMap,
+    last_render: Instant,
+}
+
+#[cfg(feature = "integration")]
+fn setup_integration_logging() {
+    let level = std::env::var("HELIX_LOG_LEVEL")
+        .map(|lvl| lvl.parse().unwrap())
+        .unwrap_or(log::LevelFilter::Info);
```

Comment:
It would be nice if we can have log env kind like of filter rather than showing everything. Like `RUST_LOG=helix-term:info`.

Reply:
Yeah I thought about that too, but that would mean introducing a second logger.